mmv1/third_party/terraform/website/docs/r/google_service_account.html.markdown

```diff
@@ -15,7 +15,8 @@ Allows management of a Google Cloud service account.
 * How-to Guides
     * [Official Documentation](https://cloud.google.com/compute/docs/access/service-accounts)
 
--> **Warning:**  If you delete and recreate a service account, you must reapply any IAM roles that it had before.
+-> **Warning:**  If you delete and recreate a service account, you must reapply any IAM roles that it had before. Also don't use this resource to 
```

Comment:
Please revert this documentation change, since it's not related to disabling service accounts.

Reply:
Let's leave it in. It is an important note related to one of the linked issues. I'm a big fan of tacking on doc changes to PRs.

Reply:
Hi upodroid - sorry, but I have to disagree. I prefer to have PRs be contained to just the issue at hand. Also, although I agree that this is a good point, I don't think this is necessarily the best way to document it. And I don't want this PR to get tied up in discussing the best way to document the intricacies of the different types of service account resources / data sources.

Please revert this change.

Reply:
I have reverted it.